vm/integer.go

```diff
@@ -383,6 +383,25 @@ func builtinIntegerInstanceMethods() []*BuiltinMethodObject {
 				}
 			},
 		},
+		// Returns the `Decimal` conversion of self.
+		//
+		// ```Ruby
+		// 100.to_d # => '100'.to_d
+		// ```
+		// @return [Decimal]
+		{
+			Name: "to_d",
+			Fn: func(receiver Object, sourceLine int) builtinMethodBody {
+				return func(t *thread, args []Object, blockFrame *normalCallFrame) Object {
+					if len(args) > 1 {
```

Comment:
why does it receive one argument? I don't see we use any here

Reply:
I should've mentioned about https://github.com/goby-lang/goby/issues/648

Reply:
I think either you also implement the feature that requires the argument, or you set the limit to 0 until you're going to implement that